packages/Main/src/Core/Prefab/PlanarView.js

```diff
@@ -20,6 +20,7 @@ class PlanarView extends View {
      * @param {Extent} extent - The ground extent.
      * @param {Object} [options] - See options of {@link View}.
      * @param {boolean} [options.noControls=false] - If true, no controls are associated to the view.
+     * @param {boolean} [options.dynamicCameraNearFar=true] - The camera's near and far are automatically adjusted.
```

Comment:
Just a detail: the doc would be better organized if this option appeared right before `farFactor` and `fogSpread` which are related to dynamic near/far.